src/renderer/widgets/WalletDetails/model/vault-details-model.ts

```diff
@@ -0,0 +1,38 @@
+import { createStore, createEvent, forward, createEffect, sample } from 'effector';
+
+import type { ShardAccount, Chain, ChainId } from '@shared/core';
+import { chainsService } from '@entities/network';
+
+const shardsSelected = createEvent<ShardAccount[]>();
+const shardsCleared = createEvent();
+
+const chainSetFx = createEffect((chainId: ChainId): Chain | undefined => {
+  return chainsService.getChainById(chainId);
```

Comment:
Get chain by id it sync function, is it really needed to use it in `effect`, not in `fn`?

Reply:
One day we will create `chains model` and use chains in source
Right now I think it's more correct to call services inside effects, because they do some underground work like network API call
ChainsService only access `chains.json` file but still, good to have all side effects from services in effects